datalad/core/local/create.py

```diff
@@ -347,7 +347,10 @@ def __call__(
         cfg_proc_specs = []
         if cfg_proc:
             discovered_procs = tbds.run_procedure(
-                discover=True, result_renderer='disabled')
+                discover=True,
+                result_renderer='disabled',
+                return_type='list',
```

Comment:
why not to have it a generator here ? it seems to be used only in a single `for` below so should be ok and might "discover" faster (before exploring all possible locations for the procs)

Reply:
not much reason, I'll change it :+1:

Reply:
changed and force-pushed